tests/Checker/Preparations/Force_Single_Plugin_Preparation_Tests.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Tests for the Force_Single_Plugin_Preparation class.
+ *
+ * @package plugin-check
+ */
+
+namespace Checker\Preparations;
+
+use WordPress\Plugin_Check\Checker\Preparations\Force_Single_Plugin_Preparation;
+use WP_UnitTestCase;
+use Exception;
+
+class Force_Single_Plugin_Preparation_Tests extends WP_UnitTestCase {
+
+	public function test_prepare_plugin_exists() {
+
+		$preparation = new Force_Single_Plugin_Preparation( 'akismet/akismet.php' );
+		$message     = '';
+
+		try {
+			$preparation->prepare();
+		} catch ( Exception $e ) {
+			$message = $e->getMessage();
+		}
+
+		$this->assertEquals( 'The plugin akismet/akismet.php does not exists', $message );
```

Comment:
Instead of this, we should use `expectException`, something like this:
```php
$this->expectException( 'Exception' );
$this->expectExceptionMessage( ... );
$preparation->prepare();
```

Reply:
Updated this now, thank you!